RELEASE_NOTES.md

```diff
@@ -8,6 +8,7 @@
 - A new FileSystemResourceLocker has been added. It allows for true threadsafe locking without external dependencies.
 - The CSS can now run multithreaded with multiple workers, this is done with the `--workers` or `-w` flag.
 - When starting the server through code, it is now possible to provide CLI value bindings as well in `AppRunner`.
+- Introduced a GenericTemplateEngine to simplify the use of templates (by dynamically assigning the right type of template engine based on the template extension).
```

Comment:
"New features" is mostly for things that are relevant for things that are relevant for end-users of the servers. This change is not really relevant for them.